From 5363b4a35a7941897a544278d0c3efa8e48dc4a9 Mon Sep 17 00:00:00 2001
From: Kyle Manna <kyle.manna@fuel7.com>
Date: Wed, 2 Nov 2011 17:26:13 +0530
Subject: [PATCH 0638/1092] mfd: TPS65910: Add I2C slave address macros

Add I2C slave addresses to the header file so that platform definitions
can use them.

Signed-off-by: Kyle Manna <kyle.manna@fuel7.com>
Signed-off-by: Vaibhav Hiremath <hvaibhav@ti.com>
---
 include/linux/mfd/tps65910.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/include/linux/mfd/tps65910.h b/include/linux/mfd/tps65910.h
index 8bf2cb9..8744ae2 100644
--- a/include/linux/mfd/tps65910.h
+++ b/include/linux/mfd/tps65910.h
@@ -21,6 +21,10 @@
 #define TPS65910			0
 #define TPS65911			1
 
+/* I2C Slave Address 7-bit */
+#define TPS65910_I2C_ID0 0x12 /* Smart Reflex */
+#define TPS65910_I2C_ID1 0x2D /* general-purpose control */
+
 /* TPS regulator type list */
 #define REGULATOR_LDO			0
 #define REGULATOR_DCDC			1
-- 
1.7.11.2

